### DISCUSSION TITLE
Methods to configure waiting time to pop lsp suggestions

### DISCUSSION THREAD

**Discussion by YtihalX** [helix-editor/helix#7553] [Q&A]
Recently I'm writing Julia using helix, it uses the word "end" instead of brace, so I heavily rely on code snippets to write functions and conditional statements, it was then I realized that the speed for helix to pop up suggestions and code snippets are utterly slow, but the diagnosis does come out pretty fast.
so I don't think this is some sort of bug, perhaps is implemented with intention.
Are there any methods to make it faster?

**Comment by ghost** [comment]
You mean [idle-timeout](https://docs.helix-editor.com/configuration.html#editor-section)?

> `idle-timeout` — Time in milliseconds since last keypress before idle timers trigger. Used for autocompletion, set to 0 for instant

```
[editor]
idle-timeout = 0
```
